backend/services/services.py

```
@@ -1,24 +1,9 @@
-from models.user import User
+from typing import Optional
 from models.post import Post, PostMedia, PostDocument, Event, Like, Comment
-from schemas.post import PostResponse, MediaPostResponse, DocumentPostResponse, EventResponse, TextPostUpdate
-from database.session import SessionLocal
-from zoneinfo import ZoneInfo
-from sqlalchemy.orm import Session, joinedload
-import shutil
-from core.connection_crud import get_connections
-from crud.notification import create_notification
-from AI.moderation import moderate_text
+from sqlalchemy.orm import Session
```

Ruff (0.8.2)

Redefinition of unused `Session` from line 3

Remove definition: `Session`

(F811)